_faq/high_involvement_status.md

```diff
@@ -0,0 +1,11 @@
+---
+question: "What is the meaning of High Involvement status of any project?"
+---
+
+It simply means that the project already has more than one proposal and if you  
+want to submit the proposal for the project, you should make sure that your proposal  
```

Comment:
remove trailing whitespace

Reply:
Actually, by two trailing spaces, I am breaking the line. If you are talking about those trailing spaces.

Reply:
lines in text files should not end with whitespace.  delete the trailing whitespace on each line.